elasticsearch/templates/_helpers.tpl

```diff
@@ -85,3 +85,25 @@ Return the appropriate apiVersion for ingress.
 {{- print "networking.k8s.io/v1beta1" -}}
 {{- end -}}
 {{- end -}}
+
+{{/*
+Returns the provided array, map values, or templatable string as an array.
+*/}}
+{{- define "elasticsearch.statefulset.toArray" -}}
```

Comment:
I had initially written this inlined in the `./statefulset.yaml` template, but @jmlrt mentioned that other places could benefit from this as well. I don't want to expand the scope of this PR, but future PRs can use this to make `extraInitContainers` etc. more flexible.